src/symbols.js

```diff
@@ -716,36 +716,76 @@ defineSymbol(text, main, spacing, "\u00a0", " ");
 defineSymbol(text, main, spacing, "\u00a0", "~");
 
 // There are lots of symbols which are the same, so we add them in afterwards.
+let i = 0;
+let ch = "";
```

Comment:
Why reuse variables?

Reply:
Why not?

Reply:
Also, the style guide in the KaTeX Contributing page instructs us to: "declare variables in the outermost scope that they are used".

Reply:
This kind of re-use of variables generally makes a compiler's job harder, because it becomes harder to tell the scope of the variable that actually matters and thus to figure out the dependencies.  I'm not familiar enough with JS JIT compilers and whether this would actually trip them up, or whether multiple variable declarations would actually cost more (without optimization, it will do more stack work).  But I share @ylemkimon's feeling that the variable `i`s and `ch`s are really different and should be declared separately...

Reply:
Okay. Revision pending.